cubed/core/ops.py

```diff
@@ -1442,3 +1443,120 @@ def smallest_blockdim(blockdims):
             m = ntd[0]
             out = ntd
     return out
+
+
+def wrapper_binop(
```

Comment:
Maybe call something like `_scan_binop` to link it to the scan implementation? I've been using a naming convention like that elsewhere in the file.